src/main/java/assignment/Reservation.java

```diff
@@ -0,0 +1,21 @@
+package assignment;
+
+public class Reservation {
+    // Assignment number 2
+    String ticketId;
+    public Reservation(String id){
+        ticketId = id;
+    }
+    public void ShowTicket(){
```

Comment:
method name should be showTicket() and not ShowTicket() with regards to methods naming convention